x-pack/plugins/infra/public/containers/logs/log_analysis/api/ml_get_jobs_summary_api.ts

```diff
@@ -54,6 +54,13 @@ const jobStateRT = rt.keyof({
   opening: null,
 });
 
+const jobAnalysisConfigRT = rt.partial({
+  per_partition_categorization: rt.type({
+    enabled: rt.boolean,
+    stop_on_warn: rt.boolean,
```

Comment:
I'm using the dev cluster

The `stop_on_warn` property might not be present
<img width="1279" alt="Screenshot 2020-09-09 at 17 03 23" src="https://user-images.githubusercontent.com/57448/92616601-b6a20180-f2be-11ea-8646-3b65f687a1b6.png">

It causes `io-ts` to crash when the anomalies page opens
<img width="1279" alt="Screenshot 2020-09-09 at 17 03 32" src="https://user-images.githubusercontent.com/57448/92616633-c0c40000-f2be-11ea-8e7c-ae6803df62f2.png">

Reply:
good catch, didn't expect that the ML API would return a partial result like this